Xamarin.Forms.Core/BorderElement.cs

```diff
@@ -1,14 +1,86 @@
-﻿namespace Xamarin.Forms
+﻿using System;
+
+namespace Xamarin.Forms
 {
 	static class BorderElement
 	{
+		const int DefaultBorderRadius = 5;
+		const int DefaultCornerRadius = -1;
+
 		public static readonly BindableProperty BorderColorProperty =
 			BindableProperty.Create("BorderColor", typeof(Color), typeof(IBorderElement), Color.Default,
 									propertyChanged: OnBorderColorPropertyChanged);
 
+		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(IBorderElement), -1d);
```

Comment:
I *think* you can go with `nameof(IBorderElement.BorderWidth)` here. please verify